src/Pretzel.Logic/Pretzel.Logic.csproj

```diff
@@ -40,6 +39,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="System.IO.Abstractions" Version="2.0.0.116" />
+    <PackageReference Include="WebMarkupMin.NUglify" Version="2.6.0" />
```

Comment:
Why don't use [`NUglify`](https://www.nuget.org/packages/NUglify/) directly?

Reply:
I think you are right, we don't need `WebMarkupMin` at all

Reply:
Fixed.